PLAN.md

```diff
@@ -0,0 +1,20 @@
+- `activeEditor.mode` isn't working correctly
+- Add `neovim` capability to activeEditor, that can be queried
+
+- Add option to hide commands in screen
+
+- Create these 'commands' in `Commands.ts`:
+
+quickOpen.show/quickOpen.show.files
+quickOpen.show.bufferLines
+quickOpen.close
+quickOpen.select.open
+quickOpen.select.openSplitHorizontal
+quickOpen.select.openSplitVertical
+
+completion.complete
```

Comment:
need a way to hook in register complete when I get to it. I haven't looked much into auto completion yet though.
I'm thinking the register name, followed by the first line
extra lines will show in what autocomplete uses for "documentation help" right now.